app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -10,6 +11,8 @@ class NPQApplicationsController < Api::ApiController
       include ApiFilter
       include ApiFilterValidation
 
+      rescue_from Identity::TransferError, with: :identity_transfer_error_response
```

Comment:
maybe we can move this to the api controller instead where the other errors are?